server/libs/domain/src/system-config/dto/system-config-ffmpeg.dto.ts

```diff
@@ -5,6 +5,9 @@ export class SystemConfigFFmpegDto {
   @IsString()
   crf!: string;
 
+  @IsString()
+  threads!: string;
```

Comment:
Sure! I can set the min to 0, but setting a max is tricky since everyone has different specs. Should `crf` also be changed to a number, or just `threads`?

Reply:
Ah, we bind this option to `string` value in the `SettingInputField`. Let's leave it as is now

Reply:
I actually went ahead and changed `threads` and `crf` to be numbers, though I had to change the `string` value to `string | number` for TS to not complain.